src/examples/blend.zig

```diff
@@ -53,29 +53,27 @@ export fn init() void {
     state.bg_pip = sg.makePipeline(pip_desc);
 
     // lot of pipeline objects for rendering the blended quads
-    pip_desc = .{
-        .shader = sg.makeShader(shd.quadShaderDesc(sg.queryBackend())),
-        .primitive_type = .TRIANGLE_STRIP,
-        .blend_color = .{ .r=1.0, .g=0.0, .b=0.0, .a=1.0 }
```

Comment:
With a trailing comma for the last argument, `zig fmt` will preserve multiple lines for `pip_desc` initialization. In other words, add a comma `,` after the closing brace `}` of `.blend_color = ...`.

```zig
.blend_color = .{ .r=1.0, .g=0.0, .b=0.0, .a=1.0 },
```